### DISCUSSION TITLE
Welcome to wt-data-project Discussions!

### DISCUSSION THREAD

**Discussion by ControlNet** [ControlNet/wt-data-project.web#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Jiaming-S** [comment]


**Reply by ControlNet** [reply]
It works fine for me. What about trying refreshing without cache?

**Comment by InnohurrytoShout** [comment]
So far with the increase of player base and less accuracy data from thunderskill, what will be the reliabilty in case of trends and number of battles?

**Reply by ControlNet** [reply]
Yeah. I see there is a sudden change in the number of battles. I guess it's because thunderskill clear the stats on that day. But now the data is accumulated and should be fine.

![image](https://user-images.githubusercontent.com/12800094/207526642-4ad352ef-60d8-4772-ad75-06d4ab9f2b6b.png)

**Comment by zarusoba10** [comment]
Is it possible to add a sort function to the table at the bottom of the page?
I want to sort the win rate, the number of battles, etc. in descending order.

**Reply by ControlNet** [reply]
It's possible, let me try.

**Reply by ControlNet** [reply]
Implemented.

**Comment by isaacvictorstatistic** [comment]
Hi, why dont have fleets ? ;O tx for graphics

**Reply by ControlNet** [reply]
The data is collected from thunderskill. As no fleet data available in thunderskill, it's no also fleet here.

**Comment by kucerp21** [comment]
Hi, it would be interesting to see some combination of ground and aviation win rates. Could be useful to see which nations are most well-rounded -> good ground and aviation vehicles at the same time.

Great site BTW. Very interesting to see these stats! 👍🏻

**Comment by z4668640** [comment]
I don't think there's been any data updates in the last few days. Did something happen? 😂

**Reply by ControlNet** [reply]
The new tier 8 ground vehicles caused some problem, but now it's fixed.

**Reply by z4668640** [reply]
If the cell is currently selected, the table below still shows no vehicles that were newly added in the last update.

**Reply by ControlNet** [reply]
@z4668640 It is because the vehicles (e.g. JAS 39) is not recorded by Thunderskill, so I cannot collect it. I have to wait the thunderskill updated.

**Reply by z4668640** [reply]
OK, I understand. Have a good day.

**Comment by svenvd-github** [comment]
Is there a reason why there is so much white space at the top of the aviation (air rb) table? 

Does for instance Sweden not have a matchmaker for **11.7 - 12.7** & **12.0-13.0**  & **12.3-13.3** while USA, USSR and Israel do? 

Does this mean playing a Swedish 12.3 (the JAS39A) always gets downtiered (or is more likely to get downtiered)?

**Reply by ControlNet** [reply]
The reason is these new vehicles are still missing in thunderskill, so it's not collected in the data yet.

**Reply by svenvd-github** [reply]
Gotcha, thanks

**Comment by IdenInTheRain** [comment]
What does the "weighted average" in the data repository mean?

**Reply by ControlNet** [reply]
For example, in one BR, there are 2 vechicles. 

Vehicle A: 1 battle, 100% winrate,
Vehicle B: 100000 battles, 50% winrate.

The weighted average is applied to calculate the overall winrate in the BR. The weight is the number of battles. Therefore, the overall winrate will be closed to 50%.

**Comment by MozzaBurger** [comment]
I've created a new post in the "issues" section just in case. Talking about some vehicles misplaced in the wrong nation tree.

Along the post I mention that ThunderSkill is under new ownership and the guy has updated a lot of the missing nations and such in case that can impact WTDP. The more accurate data the better ^^

**Comment by Nacros109** [comment]
I don't believe that this feature has been implemented yet (apologies if it has already), but I think it could be a good idea to have separate sections for each region. This way potential differences from regions can be seen - especially useful as people often favour the nation they are from. i.e the USA may suffer from worse win rates because they have a wider pool of players from varying skill (it could be that the more players there are the worse the overall skill is). Additionally, it would make the tool more informative for players from each region so they could know exactly what it is like in their server region - potentially players may switch to higher skill regions for their respective nation.
